code/game/jobs/job/support_chaplain.dm

```diff
@@ -34,6 +34,8 @@
 	var/obj/item/weapon/storage/bible/B = new /obj/item/weapon/storage/bible(H)
 
 	spawn()
+		H.equip_to_slot_or_del(B, slot_l_hand)
```

Comment:
The only problem with that is the Chaplin can change the sprite of the bible when first received. Therefore, it has to be equipped prior to the feedback cycle AND be in a location that can have is appearance updated. If there is an appearance update for the backpack slot, then that's simple. However, equipping prior to customization seems unavoidable.